core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/util/ServletUtil.java

```diff
@@ -41,6 +41,10 @@ public static boolean getRequestParamBoolean(SlingHttpServletRequest request, St
         return Boolean.parseBoolean(getRequestParamString(request, param));
     }
 
+    public static int getRequestParamInt(SlingHttpServletRequest request, String param) {
+        return Integer.parseInt(getRequestParamString(request, param));
```

Comment:
Please add the check for number via `NumberUtils`
Also, a standard good practice is to add the optional last parameter to such a method, which declares a default value for the case if number parsing failed